vocabularies/borehole-status-event.ttl

```diff
@@ -83,6 +87,24 @@ bhse:hydraulic-fracturing-completed a skos:Concept ;
     skos:prefLabel "Hydraulic fracturing completed"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/borehole-status-event> .
 
+bhse:hydraulic-fracturing-altered a skos:Concept ;
+    skos:definition "A Notice of completion of hydraulic fracturing activities but wth alteration. Lodgement of Form PGGD-04. The full well fracturing objectives were required to be changed due to unforeseen circumstances throughout the process, but hydraulic fracturing activities were carried out. This event is a recording of a reportable activity which does not change the status"@en ;
+    skos:inScheme <http://linked.data.gov.au/def/borehole-status-event> ;
+    skos:prefLabel "Hydraulic fracturing completed with alteration"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/borehole-status-event> .
+
+bhse:hydraulic-fracturing-partial a skos:Concept ;
+    skos:definition "A Notice of a partial completion of hydraulic fracturing activities. The full well fracturing objectives were not acheived, but a portion of the planned hydraulic fracturing activities were carried out. Lodgement of Form PGGD-04. This event is a recording of a reportable activity which does not change the status"@en ;
+    skos:inScheme <http://linked.data.gov.au/def/borehole-status-event> ;
+    skos:prefLabel "Hydraulic fracturing partially completed"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/borehole-status-event> .
+
+bhse:hydraulic-fracturing-related a skos:Concept ;
+    skos:definition "A Notice of a other events relating to hydraulic fracturing activities. Any other related to hydraulic fracturing such as: industrial accident/s, adverse evironmental condition/s (such as too wet, too hot, electrical storm etc.) at the wellsite, or concerns over workplace health and safety. Lodgement of Form PGGD-04. This event is a recording of a reportable activity which does not change the status"@en ;

```

Comment:
As you've used special characters, you will need to wrap this paragraph in 3 quotation marks.

Reply:
done